Applications/DataExplorer/mainwindow.cpp

```diff
@@ -1096,6 +1096,16 @@ void MainWindow::HideWindow()
 	this->hide();
 }
 
+void MainWindow::LoadFile(const QString &fileName)
```

Comment:
there is already a ```loadFile()```-method and having another one (with capital L) is confusing.
maybe rename this to ```loadFileOnStartUp()``` or something ...